src/new-client/src/app/site/[site]/users/manage/find-user-form.tsx

```diff
@@ -0,0 +1,87 @@
+﻿/* eslint-disable react/jsx-props-no-spreading */
+'use client';
+import { usePathname, useRouter, useSearchParams } from 'next/navigation';
+import React from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { EMAIL_REGEX } from '../../../../../constants';
+
+type FormFields = {
+  email: string;
+};
+
+const FindUserForm = ({ site }: { site: string }) => {
+  const searchParams = useSearchParams();
+  const pathname = usePathname();
+  const { replace } = useRouter();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FormFields>({
+    defaultValues: { email: '' },
+  });
+
+  const submitForm: SubmitHandler<FormFields> = form => {
+    const params = new URLSearchParams(searchParams);
+    if (form.email) {
+      params.set('user', form.email);
+    } else {
+      params.delete('user');
+    }
+    replace(`${pathname}?${params.toString()}`);
+  };
+
+  const cancel = () => {
+    replace(`/site/${site}/users`);
+  };
+
+  return (
+    <form onSubmit={handleSubmit(submitForm)}>
+      <div
+        className={`nhsuk-form-group ${errors.email ? 'nhsuk-form-group--error' : ''}`}
+      >
+        <div className="nhsuk-form-group">
+          {errors.email && (
+            <span className="nhsuk-error-message">
+              <span className="nhsuk-u-visually-hidden">Error:</span> You have
+              not entered a valid nhs email address
```

Comment:
Should NHS be capitalised here, as it is in other places?

Reply:
Done